src/configuration.php

```diff
@@ -542,7 +542,7 @@ class ShopgateConfig extends ShopgateContainer implements ShopgateConfigInterfac
         $this->server                         = 'live';
         $this->api_url                        = '';
         $this->shop_is_active                 = 0;
-        $this->always_use_ssl                 = 0;
+        $this->always_use_ssl                 = 0; // default should be 1, no exceptions should be made for production systems!
```

Comment:
As according to comment, this really should be "1" by default. Can't think of anything that would break by doing so.